src/containers/ClusterInfo/ClusterInfo.tsx

```diff
@@ -59,10 +75,22 @@ export const ClusterInfo = ({clusterTitle, additionalClusterInfo = []}: ClusterI
 
     const fetchData = useCallback(() => {
         dispatch(getClusterInfo(clusterName ? String(clusterName) : undefined));
+        dispatch(getClusterNodes());
     }, [dispatch, clusterName]);
 
     useAutofetcher(fetchData, [fetchData], true);
 
+    const versionToColor = useMemo(() => {
+        if (additionalVersionsProps?.getVersionToColorMap) {
+            return additionalVersionsProps?.getVersionToColorMap();
```

Comment:
In internal version we have constant versions to color mapping, so colours shouldn't be calculated